app/tasks/task.list.js

```diff
@@ -9,6 +9,18 @@ class TaskList {
     append(task){
         this.tasks.push(task)
     }
+
+    getPendingTasks(){
+        return this.tasks.filter(task => task.status === 'pending');
+    }
+
+    getStartedTasks(){
+        return this.tasks.filter(task => task.status === 'started');
+    }
+
+    getCompletedTasks(){
```

Comment:
Dont implement this code for this user story. We are implementing only "pending" task. Leave this code for its own user story (finished/completed) task.